README.md

```diff
@@ -52,13 +52,13 @@ Duffle is a command line tool that allows you to install and manage CNAB bundles
     ```console
     $ duffle bundle list
     NAME            VERSION DIGEST                                          SIGNED?
-    helloworld      0.1.0   b2747e5c36369f4c102f4f879caa94e607e5db7e        true
+    helloworld      0.1.1   b2747e5c36369f4c102f4f879caa94e607e5db7e        true
```

Comment:
Technically, the digest has changed as well, at least it looks that way for me:
```
NAME      	VERSION	DIGEST
helloworld	0.1.1  	07566b2a3822a4b8bac23ca7a5c2726ae207498b
```
Also, note, the `SIGNED?` section is no longer printed.  Should we refresh the entire output for this command?

Reply:
*sigh*, yeah that was me being lazy and just updating the version number for the sake of keeping the diff minimal. I'll update everything verbatim instead.

Reply:
@vdice I updated by copying things verbatim. I wonder if it's a little bit more verbose and if you'd like me to snip some parts of the output? Thanks for pointing out the discrepancies!